uswds/uswds-site
# USWDS-Site - POAM: May '24

# Summary

POAM updates for May 2024.

## Breaking change

This is not a breaking change.

## Related issue

Closes three dependabot alerts related to the `Nokogiri` gem

- https://github.com/uswds/uswds-site/security/dependabot/61
- https://github.com/uswds/uswds-site/security/dependabot/60
- https://github.com/uswds/uswds-site/security/dependabot/63

## Preview link

Preview link →

## Major changes

- Update bundler versions to minimize versions used in CI pipeline
- Node dependency updates
- Ruby gem updates

## Testing and review

1. Run `npm install`.
2. Run `npm run build` and confirm there are no build errors.
3. Run `npm start` and confirm there are no build errors.
4. No perceived visual regressions.

## Dependency updates

| Dependency | Previous version | New version |
| --- | --- | --- |
| axe-core | 4.9.0 | 4.9.1 |
| sass | 1.74.1 | ~1.77.0~ 1.77.2 |
| snyk | 1.1288.0 | ~1.1291.0~ 1.1291.1 |

## Ruby updates

Added `sass-embedded` to gemfile to resolve CI build errors.
